source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. ``findOne`` takes a query document, an optional sort order,
+and an optional projection. ``findOne`` returns a Promise that resolves
+to the first document matching the query in the specified sort order,
+which defaults to `natural sort order <https://docs.mongodb.com/manual/reference/glossary/#term-natural-order>`_.
+If no document matches the query, ``findOne`` returns a Promise that
+resolves to ``null``. Since ``findOne`` returns only one document, the
+Promise returned by this method resolves to an Object, not a Cursor.
```

Comment:
This is good info, but it's super focused on `findOne()` (a method) rather than explaining how to "find a document" (a use case). Could we phrase this to include more or less the same info BUT be less reference-y and instead take a more pedagogical tone as if you were explaining to someone in person how to do this?

It might be helpful to put some objectives around this (these may not be 100% but for example):

**Jasmin** should be able to:
- remember the method name
- describe how to specify a document to find (query)
- find a document in an example data set (probably out of scope)
- find more content about read operations (findOne reference, related examples, etc.)

**Lucas** should be able to:
- (Lucas already knows basic CRUD)
- explain how sort order affects the returned document
- explain how to handle errors and the "not found" case